layers/tcp.go

```diff
@@ -308,11 +308,7 @@ func decodeTCP(data []byte, p gopacket.PacketBuilder) error {
 	if err != nil {
 		return err
 	}
-	if p.DecodeOptions().DecodeStreamsAsDatagrams {
```

Comment:
Please re-add this if statement.  It allows users to decide whether they want their packets decoded beyond the TCP/UDP layer or not.  This is important in cases where, for example, someone is using port 994 for something other than ircs.